bin/install_test_requirements.sh

```diff
@@ -25,8 +25,7 @@ pip install -r https://raw.githubusercontent.com/ckan/ckanext-xloader/master/req
 # TODO: require main branch of all of these once they are updated to Python 3 and CKAN 2.11
 pip install -e git+https://github.com/opendata-swiss/ckanext-dcatapchharvest.git@feat/upgrade_to_py3#egg=ckanext-dcatapchharvest
 pip install -r https://raw.githubusercontent.com/opendata-swiss/ckanext-dcatapchharvest/feat/upgrade_to_py3/requirements.txt
-pip install -e git+https://github.com/opendata-swiss/ckanext-harvester_dashboard.git#egg=ckanext-harvester_dashboard
-pip install -r https://raw.githubusercontent.com/opendata-swiss/ckanext-harvester_dashboard/master/requirements.txt
```

Comment:
I just checked, that the requirements.txt for ckanext-harvester_dashboard still exists in the py3-branch. Maybe it makes sense to remove it entirely as we are not using it?